collector_k8s/values-statefulset.yaml

```diff
@@ -0,0 +1,93 @@
+collectors:
+  - name: statefulset
+    image: otel/opentelemetry-collector-contrib:0.58.0
+    enabled: true
+    mode: statefulset
+    replicas: 2
+    targetallocator:
+      enabled: true
+      image: ghcr.io/open-telemetry/opentelemetry-operator/target-allocator:0.1.0
+      prometheusCR:
+        enabled: false
+    scrape_configs_file: scrape_configs_statefulset.yaml
+    resources:
+      limits:
+        cpu: 2000m
+        memory: 3000Mi
+      requests:
+        cpu: 1000m
+        memory: 2500Mi
+    env:
+      - name: LS_TOKEN
+        valueFrom:
+          secretKeyRef:
+            key: LS_TOKEN
+            name: otel-collector-secret
+    config:
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+              endpoint: "0.0.0.0:4317"
+      processors:
+        memory_limiter:
+          check_interval: 1s
+          limit_percentage: 75
+        resourcedetection/gke:
+          detectors: [env, gke]
+          timeout: 2s
+          override: false
+        batch:
+          send_batch_size: 1000
+          timeout: 1s
+          send_batch_max_size: 1500
+        resource:
+          attributes:
+          - key: job
+            from_attribute: service.name
+            action: insert
+          - key: service.name
+            action: upsert
+            from_attribute: k8s.daemonset.name
+          - key: service.name
+            action: upsert
+            from_attribute: k8s.replicaset.name
+          - key: service.name
+            action: upsert
+            from_attribute: k8s.statefulset.name
+          - key: service.name
+            action: upsert
+            from_attribute: k8s.job.name
+          - key: service.name
+            action: upsert
+            from_attribute: k8s.cronjob.name
+          - key: instance
+            from_attribute: service.instance.id
+            action: insert
+          - key: k8s_cluster_name
+            from_attribute: k8s.cluster.name
+            action: insert
```

Comment:
Is there a reason for adding these attributes with different naming other than just keeping the otel semantic ones? (e.g, isn't `cloud.platform` enough?)
- instance
- k8s_cluster_name
- cloud_platform
- cloud_provider